app/src/main/java/org/jellyfin/androidtv/auth/model/AuthenticationStoreUser.kt

```diff
@@ -11,5 +11,6 @@ import java.util.*
 data class AuthenticationStoreUser(
 	val name: String,
 	@SerialName("last_used") val lastUsed: Long = Date().time,
-	@SerialName("require_password") val requirePassword: Boolean = true
+	@SerialName("require_password") val requirePassword: Boolean = true,
+	@SerialName("primary_image_tag") val primaryImageTag: String? = null
```

Comment:
```suggestion
	@SerialName("image_tag") val imageTag: String? = null
```

It's not like we're going to add more types of images so this makes more sense to me